region/maridia/inner-pink/Colosseum.json

```diff
@@ -1073,16 +1073,28 @@
           {"spikeHits": 1}
         ]},
         "h_canMaxHeightSpringBallJump",
-        "canInsaneJump",
+        {"or": [
+          "canSpringFling",
+          "canInsaneJump"
+        ]},
         "canBombJumpWaterEscape",
         "h_canJumpIntoIBJ"
       ],
       "note": [
         "Stay out of the water, and by extension the sand, of Colosseum by using the spikes as platforms.",
         "Morphing before landing on the spikes helps to be able to control the knockback.",
         "The final spike jump (which would be the most difficult) is avoided by doing a crouch jump into spring ball jump into IBJ to reach the left door.",
-        "The spring ball jump should be performed just before max height, during the 4-frame window between 2 and 5 frames before the last possible frame to jump (there are also 1-frame windows at 0 and 8 frames before the last frame).",
-        "In every case the first bomb must be placed exactly two frames before the spring ball jump."
+        "Perform the spring ball jump near max height.",
+        "Place the first bomb between about 4 and 6 frames after the spring ball jump; ideally it should be just above the water line.",
```

Comment:
I pause before placing the bomb.

Reply:
It can work, yes. But from my testing, doing this only gives a 1-frame or 2-frame window for placing the bomb, compared to a 5-frame window with pausing later. It makes sense because when pausing earlier the pause end ups hitting near the peak of the jump, where the fall speed reset doesn't do as much; by pausing later you get the fall speed reset after you've already accumulated a significant amount of fall speed, which buys more time.